docs/provisioning/internal_silos.md

```diff
@@ -1,50 +0,0 @@
-# Provisioning a Federated Learning setup with internal silos
-
-## Contents
-
-This directory contains resources to help you provision a Federated Learning setup with _internal_ silos, _i.e._ silos that are in the same Azure tenant as the orchestrator. You will be able to use this setup to run the examples in the `../../examples/pipelines` directory.
-
-For this first iteration, we are only provisioning a _vanilla_ setup, _i.e._ a setup with no security or governance features, where the silos are NOT locked down. We will add these features in future iterations. In the mean time, you should NOT be uploading sensitive data to this setup. However, you CAN use this setup for refining your training pipelines and algorithms. The only elements that will need to change when working on a _real_ secure setup will just be the orchestrator and silos names in a config file - you will be able to re-use all your code as-is.
```

Comment:
do we want to save this paragraph for another page?

Reply:
sure, here you go https://github.com/Azure-Samples/azure-ml-federated-learning/pull/72/files#diff-988a5392cf907f7c5728b8ac90445facc7f835f3c00375764ec99769dcde2d5aR21